Update cheatsheet (#1029)
AlexsLemonade/training-modules: module-cheatsheets/spatial-cheatsheet.md

- [`VisiumIO`](#visiumio)
- [`ggspavis`](#ggspavis)
- [`SpotSweeper`](#spotsweeper)
- [`Banksy`](#banksy)
- [`spacexr`](#spacexr)

> - [`scuttle`, `scran` , and `scater`](#scuttle-scran--and-scater)

Reviewer: You will need to update this given the small typo change

Author: woops, forgot doctoc

### `Banksy`

<!-- TODO: This package has not yet made it to rdrr -->
Read the [`Banksy` package documentation](https://www.bioconductor.org/packages/release/bioc/html/Banksy.html), which contains several vignettes on its usage, including a [vignette on parameter selection](https://www.bioconductor.org/packages/release/bioc/vignettes/Banksy/inst/doc/parameter-selection.html).
Note that many of their vignettes use of image-based spatial transcriptomics data that is more fine-grained compared to 10x Visium.

| Library/Package      | Piece of Code      | What it's called    | What it does  |
|----------------------|----------------------------|--------------------------------------------|--------------------------------------------------------------|
| `Banksy` | [`computeBanksy()`](https://www.bioconductor.org/packages/release/bioc/vignettes/Banksy/inst/doc/domain-segment.html#running-banksy)  | Compute `Banksy` | Compute the component neighborhood matrices for the `Banksy` matrix |

> | `Banksy`| [`runBanksyPCA()`](https://www.bioconductor.org/packages/release/bioc/vignettes/Banksy/inst/doc/domain-segment.html#running-banksy)| Run `Banksy` PCA | Run PCA on a `Banksy` matrix while weighting the spatial neighborhood context |

Reviewer: slight modification suggestion?
```suggestion
| `Banksy`| [`runBanksyPCA()`](https://www.bioconductor.org/packages/release/bioc/vignettes/Banksy/inst/doc/domain-segment.html#running-banksy)| Run `Banksy` PCA | Run PCA on a `Banksy` matrix with a weighted spatial neighborhood context |
```

> | `Banksy`| [`runBanksyPCA()`](https://www.bioconductor.org/packages/release/bioc/vignettes/Banksy/inst/doc/domain-segment.html#running-banksy)| Run `Banksy` PCA | Run PCA on a `Banksy` matrix using a provided value for the spatial weighting parameter `lambda` |

Reviewer: Do we need to say "using a provided value for the spatial weighting parameter `lambda`"? I don't think we usually call out specific parameters in the cheat sheets, and there is a default value for `lambda`. If you do want to include it, which I don't really object to, I would probably make the lambda comment a separate sentence.

Or perhaps rephrase to be more descriptive of the process without calling out the parameter? Something like "Run PCA on a `Banksy` matrix that incorporates a weighted spatial component", perhaps?

Author: >  Something like "Run PCA on a Banksy matrix that incorporates a weighted spatial component", perhaps?

I like this actually

Author: I went with a similar phrasing: https://github.com/AlexsLemonade/training-modules/pull/1029/commits/bbe90f1180d5067183cc6e9de06b24f889190a65

eh?

| Library/Package      | Piece of Code      | What it's called    | What it does  |
|----------------------|----------------------------|--------------------------------------------|--------------------------------------------------------------|
| `spacexr` | [`createRctd()`](https://www.bioconductor.org/packages/release/bioc/vignettes/spacexr/inst/doc/rctd-tutorial.html#step-1-preprocess-data)  | Create an RCTD object | Preprocess data before performing deconvolution with RCTD |
| `spacexr`| [`runRCTD()`](https://www.bioconductor.org/packages/release/bioc/vignettes/spacexr/inst/doc/rctd-tutorial.html#step-2-run-rctd)| Run RCTD | Run the RCTD algorithm to decompose cell type mixtures |
| `spacexr`| [`plotCellTypeWeight()`](https://www.bioconductor.org/packages/release/bioc/vignettes/spacexr/inst/doc/rctd-tutorial.html#single-cell-type)| Plot a single cell type's weights | Plot pixel proportions for a specific cell type on a spatial coordinates plot |

<div style="page-break-after: always;"></div>

> ### `scuttle`, `scran` , and `scater`

Reviewer: ```suggestion
### `scuttle`, `scran`, and `scater`
```
